modules/cnv-upgrading-cnv.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * cnv/upgrading-container-native-virtualization.adoc
+
+[id="cnv-upgrading-cnv_{context}"]
+= Upgrading {CNVProductName} to the next major version
+
+You can manually upgrade {CNVProductName} to the next major version by using the
+{product-title} web console to change the channel of your Operator subscription.
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Access the {product-title} web console and navigate to *Operators* -> *Installed Operators*.
+
+. Click *{CNVProductNameStart}* to open the *Operator Details* page.
+
+. Click the *Subscription* tab to open the *Subscription Overview* page.
+
+. Locate the *Channel* pane. Click the pencil icon on the right side of the
+version number to open the *Change Subscription Update Channel* window.
+
+. Select the next major version. For example, if you want to upgrade to {CNVProductName}
+{CNVVersion}, select *{CNVVersion}*.
+
+. Click *Save*.
+
+. Check the status of the upgrade by navigating to *Operators* -> *Installed Operators*.
```

Comment:
I would add something about time frames for the upgrade. Roughly 10 minutes.

Reply:
In the "about" module I have "* The amount of time an update takes to complete depends on your network
connection. Most automatic updates complete within fifteen minutes."